> The `hyprlang` commit is the same, I made sure of that. Probably why it doesn't trigger a conflict.

ok -- `nixpkgs-review` did the right thing too -- no merge conflicts.

however, the aarch64-linux build is [failing](https://logs.ofborg.org/?key=nixos/nixpkgs.278364&attempt_id=50b4a72f-6e40-4597-bbe4-3dfcc1424c92)
```console
/nix/store/wjkn3fmykn68rbpnamj9kzgw263xcga2-binutils-2.40/bin/ld: /nix/store/hddv2aciqs8k8pza6nsrhxki3ri60n1p-hyprlang-0.2.1/lib/libhyprlang.so: undefined reference to `std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >::_M_replace_cold(char*, unsigned long, char const*, unsigned long, unsigned long)@GLIBCXX_3.4.31'
```

--- a/pkgs/applications/window-managers/hyprwm/hyprpaper/default.nix
+++ b/pkgs/applications/window-managers/hyprwm/hyprpaper/default.nix
@@ -3,8 +3,10 @@
 , fetchFromGitHub
 , cmake
 , file
+, hyprlang
 , libGL
 , libjpeg
+, libwebp
 , mesa
 , pango
 , pkg-config
@@ -15,13 +17,13 @@
 
 stdenv.mkDerivation (finalAttrs: {
   pname = "hyprpaper";
-  version = "0.5.0";
+  version = "0.6.0";
 
   src = fetchFromGitHub {
     owner = "hyprwm";
     repo = finalAttrs.pname;
     rev = "v${finalAttrs.version}";
-    hash = "sha256-tcHtiyDtLky3lBk5cTmpHRSSbo1IjqOwf+q6Lofz5qM=";
+    hash = "sha256-mqxnaNiCVJS88Dk4V1v2wdS0RaCbOk8HFOUUbp0Uiy0=";
   };
 
   nativeBuildInputs = [
@@ -32,8 +34,10 @@ stdenv.mkDerivation (finalAttrs: {
 
   buildInputs = [
     file
+    hyprlang
     libGL
     libjpeg
+    libwebp
     mesa
     pango
     wayland

--- a/pkgs/top-level/all-packages.nix
+++ b/pkgs/top-level/all-packages.nix
@@ -5707,7 +5707,9 @@ with pkgs;
 
   hyprnome = callPackage ../applications/misc/hyprnome { };
 
-  hyprpaper = callPackage ../applications/window-managers/hyprwm/hyprpaper { };
+  hyprpaper = callPackage ../applications/window-managers/hyprwm/hyprpaper {
+    stdenv = gcc13Stdenv;
+  };
 
   hyprpicker = callPackage ../applications/window-managers/hyprwm/hyprpicker {
     wlroots = wlroots_0_16;

